sale_last_price_info/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Product Last Price Info - Sale",
+    "version": "16.0.1.0.0",
+    "author": "AvanzOSC, Tecnativa, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sales",
+    "license": "AGPL-3",
+    "contributors": [
```

Comment:
This is not a valid entry. Should be in CONTRIBUTORS.rst only.

If some of them (or you) want to adopt as maintainer, you should add a "maintainers":` [<github login>, ...] `instead